src/client/utils/throttle.js

```diff
@@ -0,0 +1,31 @@
+import nativeMethods from '../sandbox/native-methods';
+
+export default function throttle (func, ms) {
+    var isSuspend = false;
```

Comment:
I believe it's a little bit more complicated then that. We set cookie for particular URL, what if we throttle setting cookies for particular URL and page unload happens? I guess we should perform additional synchronization on before unload.